### PR TITLE
fix analytics partition tiers

### PR DESCRIPTION
**Pull request by seansain**

### CODE REVIEW

File: website/docs/r/partition.html.markdown [hunk 1]
```diff
@@ -24,7 +24,7 @@ The following arguments are supported:
 
 - `name` - (Required) The name of the partition.
 - `routing_expression` - (Required) The query that defines the data to be included in the partition.
-- `analytics_tier` - (Required) The Cloud Flex analytics tier for your data; only relevant if your account has basic analytics enabled. Possible values are: `enhanced`, `basic`, `cold`
+- `analytics_tier` - (Required) The Cloud Flex analytics tier for your data; only relevant if your account has basic analytics enabled. Possible values are: `enhanced`, `basic`, `cold`, `continuous`, `frequent`, `infrequent`
```
Comment (sumovishal) [inline website/docs/r/partition.html.markdown:27]: Add a note about deprecation of `enhanced`, `basic`, and `cold` values?

File: sumologic/resource_sumologic_partition.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ func resourceSumologicPartition() *schema.Resource {
 			"analytics_tier": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: validation.StringInSlice([]string{"enhanced", "basic", "cold"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"continuous", "frequent", "infrequent", "enhanced", "basic", "cold"}, false),
 				Default:      "enhanced",
```
Comment (sumovishal) [inline sumologic/resource_sumologic_partition.go:36]: Update the default value to be `continuous` to match our API docs.

**Review by sumovishal — APPROVED** [review]
lgtm, some minor comments.

File: sumologic/resource_sumologic_partition.go (hunk 2 above)
Comment (seansain) [reply sumologic/resource_sumologic_partition.go:36]: The API changes it to `enhanced`. I'm thinking maybe we should delay this PR until the API is fixed.

Comment (seansain) [reply sumologic/resource_sumologic_partition.go:36]: Or we can remove the default for now and fix that part later.

**Pull request merged**